### ISSUE TITLE
SyntaxError in generated settings.json

### ISSUE THREAD

**Issue by paulober** [raspberrypi/pico-project-generator#84]
The updated `settings.json` content in the `update_vscode_json_to_sdk1_5` branch contains a `)` where I should not be.
https://github.com/raspberrypi/pico-project-generator/blob/2ab988c000f7d5ded517ca6145845125af45e3fa/pico_project.py#L1240

**Comment by lurch** [comment]
Fixed by #85 , so can be closed?

**Comment by paulober** [comment]
@lurch Yes. Github has not closed it automatically because it waits for the feature branch to be merged into master.
